paddle/fluid/platform/gloo_context.h

```diff
@@ -0,0 +1,51 @@
+//   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
same above

Reply:
Yes, I will fix the issue in the pr: https://github.com/PaddlePaddle/Paddle/pull/26586